homeassistant/components/yamaha/media_player.py

```diff
@@ -95,7 +96,7 @@ def __init__(
         self, config: ConfigType, discovery_info: DiscoveryInfoType | None
     ) -> None:
         """Initialize the Configuration Info for Yamaha Receiver."""
-        self.name = config.get(CONF_NAME)
+        self.name = config.get(CONF_NAME, DEFAULT_NAME)
```

Comment:
Name is always set / has a default in the schema.

```suggestion
        self.name = config[CONF_NAME]
```